LIMS_IMAGE/web/src/settings.py

```diff
@@ -48,6 +48,7 @@
     'orders',
     'training',
     'laboratoryOrders',
+    'shopping',
```

Comment:
For what reason did you add this as its own app, instead of having it be part of the orders app?

Reply:
I could see this either way. Django is pretty vague about how to delineate apps, and from the [docs](https://docs.djangoproject.com/en/4.0/ref/applications/), it just defines apps as a "Python package that provides some set of features." Technically the shopping cart could be regarded as its own set of new features, or we could tie it in with orders. Personally, I'm impartial but could see how tying it in could reduce clutter in our project.

Reply:
So should we delete the shopping app and do it all in the orders app?

Reply:
I would say its up to you, but anyone else can chime in here too.
Consolidating the apps would just involve copy/pasting your code into the files in the orders app basically.